src/global/Nav.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import './Nav.css';
+
+export const Nav = () => {
+  return (
+    <div className="container-nav">
```

Comment:
AC 2,3,4 is done. 👍 As this component, `Nav` is a navigation component. What do you think about using tag semantic like [`<nav>`](https://developer.mozilla.org/es/docs/Web/HTML/Element/nav) for your reach your goal 😄 ?

Reply:
@alxmcr @JGitHub25 @Dara97 Hi! Already fix issues with this commit [5381a40](https://github.com/the-collab-lab/tcl-39-smart-shopping-list/pull/17/commits/5381a40672b763239c085f631c33e72550e36c0c). Please, let's know if we have to do something else @NataliaSacias